code/__HELPERS/cmp.dm

```diff
@@ -1,3 +1,12 @@
+/proc/cmp_epsilon_less_than(a,b,epsilon = MOLAR_ACCURACY)
+	return (a < (b + epsilon))
+
+/proc/cmp_epsilon_greater_than(a,b,epsilon = MOLAR_ACCURACY)
+	return ((a + epsilon) > b)
+
+/proc/cmp_epsilon_equals(a,b,epsilon = MOLAR_ACCURACY)
+	return (((a + epsilon) > b) && ((a - epsilon) < b))
+
```

Comment:
These procs are typically used for TimSort, our sorting proc. You're better off turning them into defines. The spirit is right tho